nxcomp/src/Loop.cpp

```diff
@@ -6498,7 +6498,7 @@ void SetTimer(int value)
 
   if (isTimestamp(lastTimer.start))
   {
-    int diffTs = diffTimestamp(lastTimer.start, getTimestamp());
+    diffTs = diffTimestamp(lastTimer.start, getTimestamp());
```

Comment:
> I'd rather use a local variable (diffTs) and rename it.

Using diffTs variable name, but dropped the global variable diffTs entirely now. Declaring diffTs in function/methods whereever needed now.